includes/admin/payments/class-refund-items-table.php

```diff
@@ -459,13 +459,12 @@ public function display_rows() {
 			</td>
 		</tr>
 		<tr id="edd-refund-submit-total" class="edd-refund-submit-line-total">
-			<td colspan="<?php echo $this->get_column_count() - 1; ?>"></td>
 
-			<td id="edd-refund-submit-button-wrapper">
-				<a id="edd-submit-refund-submit" class="disabled button-primary"><?php _e( 'Submit Refund', 'easy-digital-downloads' ); ?></a>
+			<td id="edd-refund-submit-button-wrapper" colspan="<?php echo esc_attr( $this->get_column_count() ); ?>">
```

Comment:
I think we should keep the two columns and then remove the `text-align: right;` CSS to better align the submit button. It _can_ align on larger screens but gets out of place on smaller ones with a single column:

<img width="429" alt="Screen Shot 2020-05-15 at 8 49 05 AM" src="https://user-images.githubusercontent.com/491124/82052040-f813fe80-9688-11ea-8c70-99970c9778f6.png">

Reply:
The `text-align:right;` is actually for the rows above (Refund Subtotal, Total, and taxes) and doesn't affect the button. I think one issue with the button alignment is that the spinner is hidden with `visibility:hidden;` which means it still takes up physical space in the table cell. Also, with the empty cell added back in and the button wrapper taking up one column, I end up with this:

![image](https://user-images.githubusercontent.com/4853066/82054972-cea9a180-968d-11ea-8804-2c3d20d90d3a.png)

(Note: I do have my browser zoom set to 125%, and this is not my full screen, but it is over 1200px wide. It is worse at smaller screen sizes.)

Another option might be to keep the two cells, but set the submit button to `colspan="2"` and adjust accordingly, so the button has more potential space to occupy.

Reply:
@robincornett What about keeping the bottom tablenav and moving the button there, out of the table?

Reply:
@spencerfinnell to do that, would I reintroduce the `display_tablenav` method and just override it within this class?

Reply:
@robincornett Yes, override `display_tablenav` in the Refunds list table. 

If `$which` is `top`, show nothing. Otherwise show the submit button.

https://github.com/WordPress/WordPress/blob/master/wp-admin/includes/class-wp-list-table.php#L1229-L1255

Reply:
Sounds good. I've got it moved but doing that has broken the nonce validation, so I may need help figuring that out.